src/status_im/ui/components/copyable_text.cljs

```diff
@@ -120,5 +121,5 @@
              :underlay-color colors/black
              :on-press       copy-fn
              :on-long-press  copy-fn}
-            [react/view {:background-color colors/white}
+            [react/view {:background-color background-color}
```

Comment:
so it will be nil if there is color in container ?

Reply:
No, I'm defaulting to white when no background-color is passed

Reply:
hi, did you tested the changes? I think you are assigning :background-color to wrong view. (Or this is the desired behavior?).
Also, please can you explain what are the use cases of passing background-color, means what you are trying to achieve?

https://user-images.githubusercontent.com/17097240/182918459-57540039-2c66-4da4-a72a-90a770755f14.mp4

Reply:
I now understand the issue with using `(when-not)` which would produce `nil` if background-color property was not passed to this component, I have refactored it in latest commit to use `or`
The code should now work as expected. 
It will accept the background-color passed and if there is no background-color passed it will use default value of `colors/white`

Reply:
> hi, did you tested the changes? I think you are assigning :background-color to wrong view. (Or this is the desired behavior?). Also, please can you explain what are the use cases of passing background-color, means what you are trying to achieve?
> 
>  main.mp4


Yes my primary case for introducing this change was when there were UI screens that have dark backgrounds or transparent backgrounds, example : Share tab for switcher : https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=451%3A18993 
If you add `copyable-text component` to any of the text fields on such a screen the background color is white by default which is not suitable for that use case.
I would have to pass `background-color :transparent` for it to work in that screen.

and that was my motivation to introduce this property in `copyable-text component`, I guess I should have mentioned the motivation upfront while making the PR.

Reply:
> so it will be nil if there is color in container ?



I now understand the issue with using `(when-not)` which would produce `nil` if background-color property was not passed to this component, I have refactored it in latest commit to use `or`
The code should now work as expected. 
It will accept the background-color passed and if there is no background-color passed it will use default value of `colors/white`

Reply:
@Parveshdhull  
using copyable-text component as it exists today I get this o/p
`[copyable-text/copyable-text-view {:copied-text profile-link
                                            :container-style {:background-color :transparent
                                                              :width :100%}}
          [rn/text {:style (profile-address-content (* window-width 0.7))
                    :ellipsize-mode :middle
                    :number-of-lines 1}
           profile-link]
          ]`
          
![Screenshot 2022-08-05 at 10 50 27 AM](https://user-images.githubusercontent.com/64726664/183007124-0a61e8cc-8c4a-4762-adbe-e9f9b9d5a80f.png)

 And below is the o/p after I make modification to accept `background-color` property :
 
![Screenshot 2022-08-05 at 10 53 34 AM](https://user-images.githubusercontent.com/64726664/183007282-c6d154a0-ac88-4832-85ec-95332cce09bf.png)


I think this use case is particular when text color is also white..

Reply:
hi @siddarthkay,
Thank you for the explanation and video. Yes, your provided code
```
[copyable-text/copyable-text-view {:copied-text profile-link :container-style {:background-color :transparent :width :100%}} [rn/text {:style (profile-address-content (* window-width 0.7)) :ellipsize-mode :middle :number-of-lines 1} profile-link] ]
``` 
showed desired behavior. Looks like in the above video(by me), the list item's background color was affecting the net result.
PR looks good to me, but if changes are made for the use case of switcher's copyable text, I would recommend making these changes in that PR, so that they can be properly tested. wdyt?

Reply:
> changes are made for the use case of switcher's copyable text, I would recommend making these changes in that PR, so that they can be properly tested. wdyt?


Hmmm that is also a valid explanation to keeping this change in that PR.
My initial motivation was also that since its a change to a component which can be used in other multiple PRs I moved this code to a seperate PR.

This modification also exists in the share-tab PR, I was thinking of removing it from there...
Any approach is fine by me.